stdlib/public/Windows/WinSDK.swift

```diff
@@ -71,6 +71,11 @@ public let TOOLBARW_CLASSW: [WCHAR] = Array<WCHAR>("ToolbarWindow32".utf16)
 public let TRACKBAR_CLASSW: [WCHAR] = Array<WCHAR>("msctls_trackbar32".utf16)
 public let UPDOWN_CLASSW: [WCHAR] = Array<WCHAR>("msctls_updown32".utf16)
 
+// ProcessEnv.h
+public let STD_INPUT_HANDLE: DWORD = DWORD(bitPattern: -10)
+public let STD_OUTPUT_HANDLE: DWORD = DWORD(bitPattern: -11)
+public let STD_ERROR_HANDLE: DWORD = DWORD(bitPattern: -12)
```

Comment:
These are from WinBase.h.  Please move them along with the other WinBase.h constants.

Reply:
Alright, updated to indicate they're from WinBase.h